alcs-frontend/src/app/features/application/review/decision-document/decision-document.component.scss

```diff
@@ -0,0 +1,33 @@
+@import '../../../../../styles/colors.scss';
+
+.action-btn.delete {
+  color: $error-color;
+}
+
+.table-header {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 16px;
+}
+
+.width-100 {
+  width: 100%;
+}
+
+.mat-column-action {
+  width: 8%;
+  text-align: right;
+}
+
+a {
+  font-size: 14px !important;
```

Comment:
just to clarify, we are not following 8, 16, 24 .... sizes anymore?

Reply:
The typography file has fonts all over the place, 8,9,14,16, etc

Reply:
then it will probably be later thing